tiledb/sm/array_schema/dimension.h

```diff
@@ -923,3 +920,9 @@ class Dimension {
 }  // namespace tiledb::sm
 
 #endif  // TILEDB_DIMENSION_H
+
+/** Returns the tile extent in string format. */
+std::string tile_extent_str(const tiledb::sm::Dimension& dim);
```

Comment:
Just as with `Attribute::fill_value_str()`, this function doesn't need to appear in the public interface of this class.